dagger-common/src/test/java/io/odpf/dagger/common/serde/typehandler/repeated/RepeatedPrimitiveHandlerTest.java

```diff
@@ -209,7 +211,6 @@ public void shouldReturnAllFieldsInAListOfObjectsIfMultipleFieldsPassedWithSameT
         assertArrayEquals(new String[]{"1", "2", "3"}, outputValues);
     }
 
-
     @Test
     public void shouldThrowUnsupportedDataTypeExceptionInCaseOfInCaseOfEnumForKafkaTransform() {
```

Comment:
need to change forKafka here

Reply:
Fixed as part of https://github.com/odpf/dagger/pull/140/commits/e63086b2be6d00846aaaf2e2e63019d5ebea49f9